SharePointFramework/PortfolioWebParts/src/components/ProjectList/index.tsx

```diff
@@ -151,24 +134,14 @@ export const ProjectList: FC<IProjectListProps> = (props) => {
         </div>
         <div hidden={!props.showSortBy || state.renderMode !== 'tiles'}>
           <Tooltip
-            content={
-              <>
-                Sorter flisene etter <strong>{props.sortBy}</strong>
-              </>
-            }
-            relationship={'description'}
+            content={format(strings.SortCardsByLabel, props.sortBy)}
+            relationship='description'
             withArrow
           >
             <Button
               className={styles.sortBy}
-              appearance={'transparent'}
-              onClick={() =>
```

Comment:
After [this commit](https://github.com/Puzzlepart/prosjektportalen365/pull/1253/commits/a187e826304800a9ff88ceb1776072433a8aa0d1), the onClick can't be done like this since `props.columns` is not available here.

Reply:
Beholdt det da det var i bruk for prosjektkort sorteringen. Jobber du med fiks siden du fjernet det?

Reply:
> Den er god. Jobber du med fiks, eller skal jeg?

Om det er sortering for fliser så gir det vel egentlig ikke mening å bruke columns siden det er liste-spesifikt?

Jeg kan se på det siden «jeg endret det», men du har bedre innsikt i denne webdelen nå.